Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -532,6 +532,9 @@ public void indexConcepts(String userName)
             SearchField search = field.getAnnotation(SearchField.class);
             LuceneField luceneFieldAnnotation = field.getAnnotation(LuceneField.class);
             if (search != null) {
+                if(isSearchOnDescription && search.fieldName().equals("description")){
```

Comment:
what is this if for? Shouldn't it be if `!isSearchOnDescription`?

Reply:
We shouldn't add the "description" search field when it's searchOnDescription. But we need to add the rest of the query params.

Reply:
I understand that from the code, but why?

Reply:
A redundant query will be added to the description field right? 

Like, once it will be added through this loop. Another query will be added when the "word" field is trying to build a query.

Reply:
See comment below